portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +32,15 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages = new ArrayList();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query commentsQuery = new Query("Comments");
```

Comment:
can you make "Comments" and "comments" a static final string? also - is there a reason one is capitalized and one isn't?

Reply:
"Comments" is the Task that holds all of the comments in the datastore. "comment" is a property of each entity in the Task.